frontend/src/app/patients/ManagePatients.scss

```diff
@@ -13,6 +13,10 @@
     background: inherit;
   }
 
+  .sr-patient-row-header {
+    max-width: 300px;
```

Comment:
unit should be set in rem. This is a converter https://nekocalc.com/px-to-rem-converter

Reply:
ok, will make that change! I think this is something we might want to consider addressing across the ui in the future - I noticed that we have a lot of cases where we're using px instead of rem. Might be good to standardize it.